python/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Could you check if a symlink could be used here? Generally like to avoid duplicating files... but if that turns out to be even a little bit tricky, just go ahead and merge.